### DISCUSSION TITLE
Problem displaying custom buttons

### DISCUSSION THREAD

**Discussion by einarbjorkman** [artisan-roaster-scope/artisan#961] [Q&A] [answered]
Hello, 
I would like to be able to add my own custom (Modbus write) events alongside CHARGE, DROP, and COOL END.
I have tried adding custom buttons but these only appear before clicking START. After clicking START they are replaced with the "standard" events buttons.
Now I could solve this by making sliders with the values 0-1 to have a button-like effect that would work after START, but this looks quite messy.
Any suggestions to achieve this?

Kind regards - Einar.

**Comment by roasterdave** [comment, marked as answer]
After START is pressed, is the "Buttons" item checked in the View menu?  If not, go ahead and click it.  Artisan will remember your selection.  The View selection can be set differently for OFF, ON and after START.

**Comment by einarbjorkman** [comment]
Thank you Dave, that did the trick!
